foldamers/parameters/reweight.py

```diff
@@ -284,9 +284,6 @@ def get_mbar_expectation(E_kln, temperature_list, NumIntermediates, output=None,
     if mbar == None:
         NumTemps = len(temperature_list)  # Last TEMP # + 1 (start counting at 1)
 
-        kB = unit.Quantity(
```

Comment:
This gas constant is slightly different than the one stored in simtk units (0.008314472). It looks like their reference values are a little outdated (from 2002 NIST data). https://simtk.org/api_docs/molmodel/api_docs22/Simbody/html/group__PhysConstants.html#_details. They are definitely aware of this: https://github.com/openmm/openmm/issues/2427